app/javascript/datasources/index.js

```diff
@@ -6,8 +6,35 @@ export const getPastMeetups = async () => {
     return json.data;
 };
 
-export const getJobs = async () => {
-    const result = await fetch(`${API_ROOT}/jobs`);
+export const getJobs = async (token) => {
```

Comment:
what do you think about throwing an error early here if token is nil? could avoid the extra api call